tests/framework/microvm.py

```diff
@@ -261,6 +268,8 @@ def kill(self):
                 self.memory_monitor.join(timeout=1)
             self.memory_monitor.check_samples()
 
+        self.vhost_user_blk_backends.clear()
```

Comment:
Will this call `kill` on all backends? Shouldn't this be a loop?

Reply:
ok, that's a leftover from before I replaced `__del__` with `kill`. Going to fix, thanks for spotting.

Reply:
fixed